tfe/resource_tfe_organization_token.go

```diff
@@ -68,6 +74,11 @@ func resourceTFEOrganizationTokenCreate(d *schema.ResourceData, meta interface{}
 		log.Printf("[DEBUG] Regenerating existing token for organization: %s", organization)
 	}
 
+	if err != nil {
```

Comment:
❓ how do we know here that the `err` is that the expiry is invalid? As opposed to something like the user not being authorized to create new tokens